### PR TITLE
Ignore whitespaces option added in CsvWriterOptions.

### PR DESCRIPTION
**Pull request by sd1998**
Thanks for contributing.

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

## Description

Options to ignore leading and trailing whitespaces added in CsvWriterOptions. Fix for issue #603.

## Testing

Did you add a unit test?
No.

### CODE REVIEW

**Comment by sd1998** [comment]
@lwhite1 I have fixed #603.

File: core/src/main/java/tech/tablesaw/io/csv/CsvWriteOptions.java [hunk 1]
```diff
@@ -118,6 +132,16 @@ protected Builder(OutputStream stream) {
       return this;
     }
 
+    public CsvWriteOptions.Builder ignoreLeadingWhiteSpaces(boolean ignoreLeadingWhitespaces) {
```
Comment (benmccann) [inline core/src/main/java/tech/tablesaw/io/csv/CsvWriteOptions.java]: The s should be lowercase for consistency

Comment (lwhite1) [reply core/src/main/java/tech/tablesaw/io/csv/CsvWriteOptions.java]: agreed. Otherwise, it looks great. Thank you

**Comment by sd1998** [comment]
@lwhite1 I have made the changes.

**Comment by lwhite1** [comment]
@sd1998 Thanks very much!

**Pull request merged**